README.md

```diff
@@ -0,0 +1,5 @@
+# Summer Paper 
+
+This repo holds a very important paper from the URSSI Summer School in Washington DC.
```

Comment:
This is a good description of where the paper was developed, but maybe it would be helpful to clarify the paper content as well. 

```suggestion
This repo holds a very important paper from the URSSI Summer School in Washington DC. This paper describes the importance of U isotopes to medical isotope production. 
```